Gemfile.lock

```diff
@@ -35,3 +35,6 @@ DEPENDENCIES
   fakeweb
   rspec
   ruby-pardot!
+
+BUNDLED WITH
+   1.11.2
```

Comment:
Since this `BUNDLED WITH` blurb is new in bundler 1.10, do you mind bumping the bundler dependency above to `>= 1.10`?  This should prevent some lockfile churn, if anyone else tries to bundle with an older version.